packages/app-elements/src/ui/composite/TopRightToolbar.test.tsx

```diff
@@ -0,0 +1,77 @@
+import { act, fireEvent, render, waitFor } from '@testing-library/react'
+import { TopRightToolbar, type TopRightToolbarProps } from './TopRightToolbar'
+
+const buttons = [
+  {
+    label: 'Primary',
+    size: 'small',
+    onClick: () => {
+      console.log('Primary')
+    }
+  },
+  {
+    label: 'Secondary',
+    icon: 'pulse',
+    variant: 'secondary',
+    size: 'small',
+    onClick: () => {
+      console.log('Secondary')
+    }
+  }
+] satisfies TopRightToolbarProps['buttons']
+
+const dropdownItems = [
+  [
+    {
+      label: 'Edit',
+      onClick: () => {
+        console.log('Edit')
+      }
+    },
+    {
+      label: 'Set metadata',
+      onClick: () => {
+        console.log('Set metadata')
+      }
+    }
+  ],
+  [
+    {
+      label: 'Delete',
+      onClick: () => {
+        console.log('Delete')
+      }
+    }
+  ]
+] satisfies TopRightToolbarProps['dropdownItems']
+
+describe('TopRightToolbar', () => {
+  beforeEach(() => {
+    vi.useRealTimers()
+  })
```

Comment:
is this required?

Reply:
uhm! good question! I'll remove it also from `Toolbar`!